game-core/src/main/java/games/strategy/triplea/delegate/DiceRoll.java

```diff
@@ -1182,149 +1190,6 @@ public static void sortByStrength(final List<Unit> units, final boolean defendin
     units.sort(comp);
   }
 
-  private static void sortAaSupportRules(
-      final Set<List<UnitSupportAttachment>> support,
-      final boolean defense,
-      final boolean friendly) {
-    sortSupportRules(
-        support,
-        defense,
-        friendly,
-        UnitSupportAttachment::getAaRoll,
-        UnitSupportAttachment::getAaStrength);
-  }
-
-  private static void sortSupportRules(
-      final Set<List<UnitSupportAttachment>> support,
-      final boolean defense,
-      final boolean friendly) {
-    sortSupportRules(
-        support,
-        defense,
-        friendly,
-        UnitSupportAttachment::getRoll,
-        UnitSupportAttachment::getStrength);
-  }
-
-  private static void sortSupportRules(
-      final Set<List<UnitSupportAttachment>> support,
```

Comment:
Of note' support' here is not extracted to the new calls as the sort was applied to this. Instead we'll have the callers do the sort and extract the rest of this function and variables to the new class (see first commit).